### ISSUE TITLE
The Cloud Firestore API is not available for Firestore in Datastore Mode database

### ISSUE THREAD

**Issue by EfraimLA** [quarkiverse/quarkus-google-cloud-services#422]
FAILED_PRECONDITION: The Cloud Firestore API is not available for Firestore in Datastore Mode database

Excuse me, i'm trying to authenticate to a database in datastore mode but i got this message. What is required to do this?

**Comment by EfraimLA** [comment]
Yes I know but is it a feature to develop? Is there any plan to implement this or why? What does it depend on? Thanks

**Comment by jfbenckhuijsen** [comment]
@loicmathieu close as won't fix?
